### DISCUSSION TITLE
Starting Github Desktop

### DISCUSSION THREAD

**Discussion by Mohdhafizz2** [community/community#42650] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Get running on my Windows 11

**Comment by jfullstackdev** [comment]
what help do you need ? you cannot install GitHub Desktop ? you don't know what is GitHub Desktop ?

**Comment by ghostinhershell** [comment]
Hi @Mohdhafizz2, we're happy you're here! 

You are more likely to get a useful response if you are explicit about what your project entails and what kind of help you need. Giving a few more details might help someone give you a nudge in the right direction.
